charts/seed-bootstrap/charts/vpa/templates/vpa-rbac.yaml

```diff
@@ -58,6 +58,15 @@ rules:
   - list
   - watch
   - patch
+- apiGroups:
+  - "druid.gardener.cloud"
+  resources:
+  - etcds
+  - etcds/scale
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Won't it require `patch` access? 

In addition, AFAIU, VPA looks for resource to find the topmost controller where it can recommend the resources using scale sub-resource. So, it requires get/list/watch. 
Should we harden this in such a way that, vpa-reader to have get/list/watch access over etcd/scale and vpa-actor does have access to only etcd and not etcd/scale.
Is there any document regarding specific RBAC requirement from VPA?
cc: @ggaurav10

Reply:
Currently, VPA is not working for etcd statefulset because of the same reason that it is owned by the etcd resource. What you told makes sense. I have enable get,list and watch for etcd and etcd/scale for vpa-reader. No actor updates the resources currently. So I have not added `patch`. Since, hvpa will eventually update etcd CR. I shall leave the permission to update the etcd CR to be added when hvpa shall be able to do so. Makes sense?